VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -621,22 +646,34 @@ export class ParticipantsPanelComponent implements OnInit, OnDestroy {
             ? false
             : true;
     }
-
-    private setParticipants() {
+    private updateParticipants() {
         const combined = [...this.nonEndpointParticipants, ...this.endpointParticipants];
-        combined.forEach(participant => {
-            const currentParticipant = this.participants.find(r => r.id === participant.id);
+        combined.forEach(item => {
+            const currentParticipant = this.participants.find(r => r.id === item.id);
             if (currentParticipant) {
-                if (currentParticipant instanceof LinkedParticipantPanelModel) {
-                    currentParticipant.participants.forEach(linkedParticpant => {
-                        this.updateParticipant(participant, linkedParticpant);
-                    });
-                } else {
-                    this.updateParticipant(participant, currentParticipant);
+                currentParticipant.updateParticipant(
```

Comment:
This seems to me like it will cause issues when this is LinkedParticipantPanelModel as it never passes the individual participant ids.

Reply:
Also what is the point of calling currentParticipant.updateParticipant with values that are all derived from currentParticipant this would be the same as not calling it right? 

In setParticipants where it uses updateParticipant the equivalent here would be `item.updateParticipant(....)`

Reply:
removed